go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift-knative/serverless-operator
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.5
```

Comment:
this is not going to work, as in previous PRs. We need go 1.22 and TOOLCHAIN=local for this to work.